### ISSUE TITLE
Does swift fall short of business value?

### ISSUE THREAD

**Issue by sakshimahendrusjsu** [SJSUFall2019-CMPE272/swift#3]
Slack is a powerful tool which provides built-in integrations, do companies need to read the conversations to raise support issues when slack built-in integrations allow us to create ISSUES using SLACK-BOT  without AI. 
Does swift fall short of business value?

Few references,
https://medium.com/@trevor_atx/3-new-ways-to-create-jira-issues-in-slack-3c61ed9a4247
https://api.slack.com/bot-users

 Example,

Issues created via Bot Mentions,
1. Example 1: @Jira create bug Create Issue not working in unique project BEER in pi-devs site
2. Example 2  @Jira create task Create feature flags for releasing without transition, assign, comment
3. Example : @Jira create bug When I put a long description for an issue — a button with not rounded corners is rendered
All this is done via slack-bot apis which create issues with a single click.

2. Companies have a tough time and  struggle  more with tracking and maintaining these issues                                                                                 Why do we need to track the issues?
     2.A) They help to give a real-time status of teams and builds.
     2.B) Helps to identify who is doing what
     2.C) Set Alerts for the issue 

 Slack  provides business value to organizations because it reduces the effort, time and makes the communication effective  and clear.
Will SWIFT be able to create such a business value with the overhead of reading the messages.

**Comment by mdhsieh** [comment]
Yes, there is the jira app to specifically help software companies, but it does not take the next step of bringing the issue to the correct department, which AI in the form of NER can help do. In addition, there may be times when an employee him/herself doesn’t know who the issue should go to, for example in a large company, in which case even when the issue is created the company’s support team must spend time deciding where to send it. Having the employee create the issue and assign with Slack is time-consuming on the employee’s part as well. Our app does act like a bot in Slack but automates the issue sending and alerts.
